packages/collector/test/tracing/misc/require_hook/app.js

```diff
@@ -26,11 +27,16 @@ app.get('/', (req, res) => {
   res.sendStatus(200);
 });
 
-app.get('/requireRequestPromiseMultipleTimes', (req, res) => {
-  require('request');
-  require('request-promise'); // executes stealthy-require
-  require('request-promise-native'); // executes stealthy-require
```

Comment:
_issue_: This is a very special test use case. See https://github.com/instana/nodejs/pull/71/files
I doubt your code simulates the underlying problem with stealthy-require, because node-fetch doesn't make use of the require cache. Either we remove the whole test or we need to find a way to simulate the problem on our selfs (preferred solution). I'd suggest to use stealthy-require directly in the test. You just need to choose a library you want to test the behaviour with.

Reply:
update, could please check, I used superagent libarary to test

Reply:
If you comment out https://github.com/instana/nodejs/pull/71/files#diff-107040a34b1332891e461280fb3a19011b4a0dbbbb5e8cca84e657ad28768769R44, the test should fail. Does it?

Reply:
Looks great 👍